web-api/src/storage/property_filter.rs

```diff
@@ -81,6 +86,45 @@ impl IndexedPropertiesSchema {
         self.properties.extend(schema_update.clone().into_iter());
         Ok(())
     }
+
+    pub(crate) fn validate_property(
+        &self,
+        document: &DocumentId,
+        property: &DocumentPropertyId,
+        value: &DocumentProperty,
+    ) -> Result<(), InvalidDocumentProperty> {
+        // This code only checks additional validity constraints coming from on a schema
+        // but an otherwise expect a valid property, hence why we accept a `&DocumentProperty`
+        // instead of a `&Value`.
+        let value = &**value;
+        let Some(definition) = self.properties.get(property) else {
+            return Ok(())
```

Comment:
:art: trailing comma, i wonder why clippy doesn't point that out 🤔 
```suggestion
            return Ok(());
```